Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -141,6 +141,13 @@ function Main {
 			if ($TestParam -imatch "mpi_type") {
 				$MpiType = [string]($TestParam.Replace("mpi_type=", "").Trim('"'))
 			}
+			if ($TestParam -imatch "install_ofed_from_extension") {
+				if($TestParam.Split("=")[1] -match "yes") {
```

Comment:
While not necessary for this PR you guys should work on improving the framework. Checking for the presence of a testparam and it's value should be available to all tests and done in functions rather than explicitly everywhere.
For example this should read better like:
$installOFEDFromExtension = (TestParamIsSet "install_ofed_from_extension")

Reply:
Also this should only happen if INFINIBAND_INSTALL_OFED=yes.
i.e. in pseudo-code
$installOFEDFromExtension = (INFINIBAND_INSTALL_OFED=yes) and (INFINIBAND_INSTALL_OFED_FROM_EXTENSION=yes).

Reply:
INFINIBAND_INSTALL_OFED=yes means install OFED driver from http://content.mellanox.com/ofed/MLNX_OFED-4.5-1.0.1.0/MLNX_OFED_LINUX-4.5-1.0.1.0-rhel
INFINIBAND_INSTALL_OFED_FROM_EXTENSION=yes => install OFED from extension